testutil/integration/router.go

```diff
@@ -146,3 +148,14 @@ func (app *App) Context() context.Context {
 func (app *App) QueryHelper() *baseapp.QueryServiceTestHelper {
 	return app.queryHelper
 }
+
+// SetMultiStore is a helper for setting up multiple stores for provided modules.
+func SetMultiStore(keys map[string]*storetypes.KVStoreKey, logger log.Logger) storetypes.CommitMultiStore {
```

Comment:
With `Set` I actually expect it more to be stored in a struct or something. Maybe `Create` is a better word?